### PR TITLE
Chainport changes

### PR DESCRIPTION
**Pull request by Ignacio-Freire**

### CODE REVIEW

File: contracts/Interfaces/ILosslessGovernance.sol [hunk 1]
```diff
@@ -36,6 +36,9 @@ interface ILssGovernance {
     function claimCommitteeReward(uint256 _reportId) external;
     function setCompensationAmount(uint256 _amount) external;
     function losslessClaim(uint256 _reportId) external;
+    function setRevShareAdmin(address _address) external;
```
Comment (DomantasP) [inline contracts/Interfaces/ILosslessGovernance.sol:39]: Maybe let's make share lowercase for consistency

File: contracts/LosslessGovernance.sol [hunk 2]
```diff
@@ -27,6 +27,8 @@ contract LosslessGovernance is ILssGovernance, Initializable, AccessControlUpgra
 
     uint256 public compensationPercentage;
 
+    uint256 public revsharePercent;
```
Comment (DomantasP) [inline contracts/LosslessGovernance.sol:30]: Contract is upgradable, this would break stuff on the deployed testnets.

**Review by DomantasP — CHANGES_REQUESTED** [review]

File: contracts/Interfaces/ILosslessGovernance.sol (hunk 1 above)
Comment (Ignacio-Freire) [reply contracts/Interfaces/ILosslessGovernance.sol]: Addressed on new commits

File: contracts/LosslessGovernance.sol (hunk 2 above)
Comment (Ignacio-Freire) [reply contracts/LosslessGovernance.sol]: Addressed on new commits

**Pull request merged**